terrain_test.py

```
@@ -0,0 +1,86 @@
+#!/usr/bin/env python3
+"""
+Visual test for the terrain generation.
+"""
+
+import curses
+import time
```

Ruff (0.8.2)

`time` imported but unused

Remove unused import: `time`

(F401)